### ISSUE TITLE
MISP.log_skip_db_logs_completely shows error log in server settings even if set to false

### ISSUE THREAD

**Issue by cudeso** [MISP/MISP#5502]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | Ubuntu 18.0.4
| OS version (client)       | 
| PHP version               | 7.2
| MISP version / git hash   | 2.4.119
| Browser                   | 

### Expected behavior
In Server Settings, MISP settings: 

Diplay no error message if MISP.log_skip_db_logs_completely is set to False

### Actual behavior
If MISP.log_skip_db_logs_completely is set to False, the Error message column still shows the error message (Logging has now been disabled - your audit logs will not captur ...) . The line is not coloured. It's a minor UI inconvenience.

Similar to https://github.com/MISP/MISP/issues/5394

### Logs, screenshots, configuration dump, ...
![2020-01-13 16_36_32-Servers - MISP](https://user-images.githubusercontent.com/256028/72269162-68e7d300-3623-11ea-908b-f19514f66940.png)

**Comment by iglocska** [comment]
Yeah it's a known issue, need to find some time to fix it, it's catching me off guard way too often too. Basically if the entry is not red/yellow/green it's not erroring.
